bot/app/__init__.py

```diff
@@ -81,6 +82,28 @@ def send_template(u_id, template):
 			parse_mode='html'
 		)
 
+def keys(status):
```

Comment:
Are we using this function somewhere?

Reply:
actually, no. it is a stub from going down different paths in ux design: i aimed to provide a markup keyboard for pushing messages. was hesitant to delete it, since we might need that some time soon